engine/test/integration_tests/dm_many_tables/conf/job.yaml

```diff
@@ -12,6 +12,7 @@ upstreams:
       password: ''
     source-id: mysql-01
     block-allow-list: balist-01
+    loader-thread: 1
```

Comment:
Why adding this config item

Reply:
without this config lightning runs too fast so we can only see 0% directly jumped to finished 😂